hana/extract_hana_package.sls

```diff
@@ -0,0 +1,47 @@
+{%- from "hana/map.jinja" import hana with context -%}
+
+{% if hana.hana_archive_file is defined %}
+{% set hana_package = hana.hana_archive_file %}
+{% set hana_extract_dest = hana.hana_extract_dir %}
+
+setup_hana_extract_directory:
+    file.directory:
+      - name: {{ hana_extract_dest }}
+      - mode: 755
+      - makedirs: True
+
+{% if (".ZIP" in hana_package) or (".zip" in hana_package) or (".RAR" in hana_package) or (".rar" in hana_package) %}
```

Comment:
Maybe better:
```{%- if hana_package.endswith((".ZIP", ".zip", ".RAR", ".rar")) %}```
What do you think?

Reply:
Yes thats better, I tested it and works fine